Documentation/ImportantOnly_NLog.config

```diff
@@ -40,9 +37,6 @@
     <logger name="Stratis.Bitcoin.Consensus.*" minlevel="Debug" writeTo="debugConsensusFile" />
     <logger name="Stratis.Features.FederatedPeg.Collateral.CheckCollateralFullValidationRule" minlevel="Debug" writeTo="debugConsensusFile" />
 
-    <logger name="Stratis.Bitcoin.Consensus.ChainedHeaderTree" minlevel="Debug" writeTo="debugCHTandCMFile" />
-    <logger name="Stratis.Bitcoin.Consensus.ConsensusManager" minlevel="Debug" writeTo="debugCHTandCMFile" />
-
```

Comment:
Perhaps remove the rules in line 52/53 as well (can't directly add the comment there unfortunately), then the CMB file will be a lot slimmer.

Reply:
I figured that might be useful to keep? Just in terms of lining up the timelines with the other components

Reply:
Ok, fair enough